### ISSUE TITLE
Webroot parameter

### ISSUE THREAD

**Issue by yamlfd** [ebekker/ACMESharp#190]
I guess I have a more complex configuration here and I do not like to see the script changing my IIS.

1. On a fileserver with Central Certificate Store I'm running the PS cmdlets.
2. These need to place the request files into a public visible folder.
3. I will add virtual directorys ".well-known" to all sites that point to these UNC path.
4. Once the scripts placed the files they can be verified.

I cannot find a parameter to place the request files into a custom folder. How can this done?

**Comment by ebekker** [comment]
You can run this cmdlet to see the list of parameters:
```PowerShell
Get-ACMEChallengeHandlerProfile -GetChallengeHandler iis -ParametersOnly
```

You'll see there is a parameter there `OverrideSiteRoot` that allows you to specify the path to where you want the generated `.well-known` files to be placed.

However, there is a bug in the current released version (0.8.1) of ACMESharp PS module that actually conflicts that parameter with another parameter to skip the local Web config.  The next release .0.8.2 fixes this but it won't be released for another week or two.  If you have an urgent situation you can pull one of the latest builds from the STAGE PowerShell Nuget repo:

https://www.myget.org/gallery/acmesharp-posh-staging

**Comment by yamlfd** [comment]
Thanks for your help. Look into it. But yes the other bug blocks everything :-(((

**Comment by WallyRanson** [comment]
If this issue is still not solved is it then possible for you to use the HTTP redirect feature of LE instead?
Read "Picking a Challenge Type" and "Central Validation Servers" on https://letsencrypt.org/docs/integration-guide/

This should make the virtual directories unneeded.

**Comment by ebekker** [comment]
Yes, you can setup the redirect in IIS or within an application of your that you host on IIS to redirect.

**Comment by alexhass** [comment]
It might be misunderstand, but I do NOT like to use any IIS handler at all. There is no need to do this and I do not want your script to break my server, so - simply I need a way to save the files into a folder without involving IIS. I may also use the script on a file server that has no IIS installed.

**Comment by WallyRanson** [comment]
Look into this thread.
https://github.com/ebekker/ACMESharp/issues/240

Its my script and I write the cert files to a directory and then move them to a separate server for use in other programs, like spam gateway, web services and exchange.
Its probably something like that you want.
